_posts/12-05-01-Building-your-Application.md

```diff
@@ -24,33 +24,33 @@ Među zadacima koje biste možda želeli da automatizujete su:
 ### Alati za automatizaciju izgradnje
 
 Alati za izgradnju mogu biti opisani kao skup skripti koje rukuju zajedničkim zadacima razvoja softvera. Alat za
-zgradnju nije deo vašeg softvera, on deluje na vaš softver 'izvana'.
+izgradnju nije deo vašeg softvera, on deluje na vaš softver 'izvana'.
 
 Postoji mnogo dostupnih alata otvorenog koda da vam pomogne sa automatizacijom, neki su napisani u PHP-u, drugi nisu.

```

Comment:
Da, da, to je mnogo bolje. 

btw Ja sam se do sada uvek trudio da "kod" pišem "kôd", dakle za "o" sam koristio dijakritički karakter, upravo da bi se znalo na šta se misli. :smiley: Prečica za taj karakter je Alt + 147.